app/models/mechanic.rb

```diff
@@ -0,0 +1,4 @@
+class Mechanic < ApplicationRecord
+  has_many :ride_mechanics
+  has_many :rides, through: :ride_mechanics
+end
```

Comment:
A note for the AmusementPark model --> putting this comment here because you don't have changes in that file so it's not showing up

For AmusementPark, you have `has_many :rides` so, for US 3, you could extend associations further to make things easier for yourself and add `has_many :mechanics, through: :rides` as an additional association. Then, this allows you to take an amusement_park instance and do `amusement_park.mechanics.distinct` in a model method to complete US 3